src/debug.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include <string>
+#include <iostream>
```

Comment:
Why do we need this? We want to apply the zero-overhead philosophy which tells us only to include the thing we need.

Reply:
I got an error about std::cerr not being defined when I tried to use DEBUG_MSG_RED in dijkstralearning without iostream included.

Reply:
I didn't get the error somehow. Wired. There must be some inconsistency in our building system. Let's use this include here. And find out later how we get inconsistent compile errors.